packages/smooth_app/lib/pages/product/simple_input_text_field.dart

```diff
@@ -40,19 +40,16 @@ class SimpleInputTextField extends StatelessWidget {
                 optionsBuilder: (final TextEditingValue value) async {
                   final String input = value.text.trim();
 
-                  if (input.isEmpty) {
```

Comment:
This will be great for packaging shapes and packaging materials, in order to have the most popular suggestions available without having to type the first letter. But we probably should not do it for other tagType values than packaging_shapes and packaging_materials, as it probably would be confusing for things like categories and labels. Especially as currently we return suggestions in alphabetic order for them.

Reply:
Also addressed by @monsieurtanuki in #3750

Reply:
The solution provided in #3750 is indeed better, I removed mine from this PR.